maisim/maisim.Game/Graphics/UserInterface/MainMenuButton.cs

```diff
@@ -1,106 +1,119 @@
+                Anchor = Anchor.Centre,
+                Origin = Anchor.Centre,
+                Size = new Vector2(400, 60),
+                Masking = true,
+                CornerRadius = 30,
+                BorderThickness = 5,
+                BorderColour = Color4.White,
+                Children = new Drawable[]
+                {
+                    new Box
+                    {
+                        RelativeSizeAxes = Axes.Both,
+                        Anchor = Anchor.CentreRight,
+                        Origin = Anchor.CentreRight,
+                        Colour = buttonColor
+                    },
+                    new GridContainer
+                    {
+                        RelativeSizeAxes = Axes.Both,
+                        ColumnDimensions = new[]
+                        {
+                            new Dimension(GridSizeMode.Absolute, 100),
+                            new Dimension(GridSizeMode.Absolute, 400)
+                        },
+                        Content = new[]
+                        {
+                            new Drawable[]
+                            {
+                                new Container
+                                {
+                                    Anchor = Anchor.CentreLeft,
+                                    Origin = Anchor.CentreLeft,
+                                    Size = new Vector2(100, 70),
+                                    Children = new Drawable[]
+                                    {
+                                        new SpriteIcon
+                                        {
+                                            Anchor = Anchor.Centre,
+                                            Origin = Anchor.Centre,
+                                            Icon = buttonIcon,
+                                            Size = new Vector2(30),
+                                            Colour = Color4Extensions.FromHex("ffffff")
+                                        }
+                                    }
+                                },new Container
+                                {
+                                    Anchor = Anchor.CentreLeft,
+                                    Origin = Anchor.CentreLeft,
+                                    Size = new Vector2(400, 70),
+                                    Child = new MaisimSpriteText()
+                                    {
+                                        Anchor = Anchor.CentreLeft,
+                                        Origin = Anchor.CentreLeft,
+                                        Text = buttonText,
+                                        Font = MaisimFont.GetFont(size: 35f)
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            };
+
+            drawableHoverSample = new DrawableSample(sampleStore.Get("hover.wav"));

```

Comment:
Sound feedback upon hover logic could be moved to a `ClickHoverSounds` drawable component to allow reuse by all interactive ui components